app/components/paperShow/noteForm/index.tsx

```diff
@@ -29,54 +93,76 @@ class PaperNoteForm extends React.PureComponent<PaperNoteFormProps, PaperNoteFor
 
     this.state = {
       note: props.initialValue || "",
+      isBlocking: false,
+      isFocus: false,
     };
   }
 
   public render() {
-    const { isLoading, handleCloseDropdown } = this.props;
-    const { note } = this.state;
+    const { isLoading, autoFocus, hideButton, omitCancel, textareaStyle, textAreaClassName, row, isEdit } = this.props;
+    const { note, isBlocking, isFocus } = this.state;
 
     return (
-      <form style={{ borderRadius: "8px" }} onSubmit={this.handleSubmit}>
+      <form
+        style={{
+          border: isFocus ? "solid 3px #e7eaef" : "solid 1px #e7eaef",
```

Comment:
Please describe me why this component could not catching its state with ':focus'.

Reply:
Cuz one of the target to be styled is a parent of this node.

As you know, almost every CSS selector doesn't support to select parent(upper node).
We could use `:focus-within` pseudo selector, however, it doesn't support IE & Edge.